k8s/helmfile/env/production/argocd-config.values.yaml.gotmpl

```diff
@@ -1,2 +1,2 @@
 environment: production
-
+appOfAppsVersion: 1.1.0
```

Comment:
maybe set this to 1.1.1 anyway since I think it's unused until we for to `argocd-config` `1.1.0` anyway? Otherwise I bet we forget?

Reply:
valid concern but due to coordination difficulties I will move on now to see if this even finally works as intended